.github/workflows/publish.yml

```diff
@@ -7,6 +7,8 @@ on:
   push:
     branches:
       - "eve-kernel-amd64-v5.10.186-generic"
+env:
+  branch_name: "refs/heads/eve-kernel-amd64-v6.1.38-generic"
```

Comment:
This should be `eve-kernel-amd64-v5.10.186-generic`

Reply:
@yash-zededa  Good catch! Fixed!